src/test/java/com/ecse437/project/ProjectApplicationTests.java

```diff
@@ -1,13 +1,85 @@
 package com.ecse437.project;
 
+import static org.assertj.core.api.Assertions.assertThat;
+
 import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
 
+import com.ecse437.project.common.Common.Algorithm;
+import com.ecse437.project.controller.ProjectController;
+import com.ecse437.project.exceptions.IllegalAlgorithmException;
+import com.ecse437.project.services.Binary;
+import com.ecse437.project.services.Factorial;
+import com.ecse437.project.services.Hexa;
+import com.ecse437.project.services.Main;
+import com.ecse437.project.services.Prime;
+import com.ecse437.project.services.Squared;
+
 @SpringBootTest
 class ProjectApplicationTests {
```

Comment:
All these tests should be refactored into different classes. We especially want to separate integration and unit tests